### DISCUSSION TITLE
Mercadopago types [@types/mercadopago]

### DISCUSSION THREAD

**Discussion by KoddyDev** [DefinitelyTyped/DefinitelyTyped#62132] [Issues with a @types package]
I saw here and the object "ipn" was not typed in the typing '@types/mercadopago'

[a comment by an automated account is not shown]

**Comment by danieldspx** [comment]
> I saw here and the object "ipn" was not typed in the typing '@types/mercadopago'

Feel free to open a PR adding the types. Also you should link the documentation supporting your changes.
